ios/RNAppAuth.m

```diff
@@ -205,7 +209,7 @@ - (void)authorizeWithConfiguration: (OIDServiceConfiguration *) configuration
                                                        strongSelf->_currentSession = nil;
                                                        if (authState) {
                                                            resolve([self formatResponse:authState.lastTokenResponse
-                                                               withAdditionalParameters:authState.lastAuthorizationResponse.additionalParameters]);
+                                                               withAdditionalParameters:authState.lastTokenResponse.additionalParameters]);
```

Comment:
I could use some additional thoughts and opinions on this change. What is the reasoning behind returning the `additionalParameters` from the authorization response instead of the token response? Dropbox returns the UID in the token response, so this change allowed me to pull that out. Otherwise there is no way to access it.